streamlit_app/app.py

```
@@ -0,0 +1,122 @@
+
+with st.sidebar:
+    st.header("Options")
+    save_png = st.checkbox("Also save PNGs to disk (output/)", value=False)
+    download_zip = st.checkbox("Show Download ZIP button", value=True)
+    min_doubletext_threshold = st.number_input("Min double-text length to show", min_value=2, max_value=20, value=2)
+
+if uploaded is not None:
+    try:
+        raw_bytes = uploaded.read()
+        text = raw_bytes.decode("utf-8")
+    except Exception:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)